src/UI/InputHandler.java

```diff
@@ -1,22 +1,26 @@
 package UI;
 
 import java.awt.event.KeyEvent;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileWriter;
-import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
-import java.util.InputMismatchException;
+import java.util.List;
 import java.util.Scanner;
-
+import Engine.ConfigUtils;
 import Engine.IController;
 import Engine.OptionSelector;
+import Engine.Pair;
 
 public class InputHandler implements IController
 {
   private static final String KEYS_FILENAME = "res/keys.txt";
+  private static final String OTHER_INPUT_FILENAME = "res/other_input.txt";
+
+  public static GameOptionBool seekBuildingsLastOption = new GameOptionBool("Seek Buildings Last", true);
```

Comment:
Still not sure this is the best phrasing. Maybe "Seek Units First", or "SEEK Prioritizes Units", but that's a little verbose.

Or maybe have the item name as "SEEK", and the options be "Units then Buildings" or "Units with Buildings", and maybe even have a "Units Only" option for the real old-skool folks (though I don't really see a need for that last one).